Source/ACE/Entity/Player.cs

```diff
@@ -1390,16 +1406,57 @@ public ActionChain GetSaveChain()
         }
 
         /// <summary>
+        /// This method is used to clear the wielded items list ( the list of ace objects used to save wielded items ) and loads it with a snapshot
```

Comment:
I would still prefer all the Snapshop methods accept a boolean indicating whether or not to clear the dirty flags.  It's a bit of a trap call right now that could cause data loss down the road if called incorrectly for whatever reason.

Reply:
Done, but not using this yet - I put this in but put a default parameter of false to mimic current behavior.   When you have time, can you define when we would want to use that vs when it is handled behind the scenes.